### ISSUE TITLE
[BUG] Plus button not show while allowAddFolder(true) set in older Android device.

### ISSUE THREAD

**Issue by eyesblue** [codekidX/storage-chooser#116]
Hi, thanks for your great project, it really simple and easy to use.

I like show a directory chooser and allow user create new directory in my app, I modify and test the example code in Xiao Mi A1(Android 9, API level28), it work well, then I create emulator for Android 4.4(API level 19 with Google APIs) and Android 4.1(API level 16 with Google APIs), it not show the plus button in the older version no matter internal or external storage, is the API not support older version?

this is my code:
---
StorageChooser chooser = new StorageChooser.Builder()
     .withActivity(StorageManageActivity.this)
     .withFragmentManager(getFragmentManager())
     .withMemoryBar(true)
     .allowCustomPath(true)
     **.allowAddFolder(true)**
     **.setType(StorageChooser.DIRECTORY_CHOOSER)**  // For pickup folder.
     .withContent(c)
     .build();

     chooser.show();

chooser.setOnSelectListener(new StorageChooser.OnSelectListener() {
     @Override
     public void onSelect(String path) {
           Toast.makeText(StorageManageActivity.this, path, Toast.LENGTH_LONG).show();
           Log.e("SELECTED_PATH", path);
     }
});

---

![Screenshot_20200101-074138](https://user-images.githubusercontent.com/11750590/71636680-776edb80-2c6e-11ea-89ad-724e09f62367.png)
The Screenshot of Xiao Mi A1, API level 28

![Screenshot_1577835640](https://user-images.githubusercontent.com/11750590/71636676-66be6580-2c6e-11ea-941a-dba567972a4e.png)
The Screenshot of Emulator, API level 16 and 19

**Smartphone :**
 - Device: 
1. Xiao Mi A1(Android 9, API level28), work fine.
2. Android studio Emulator: 4.4(API level 19 with Google APIs), x86 image.
3. Android studio Emulator: 4.1(API level 16 with Google APIs), x86 image.

**Comment by codekidX** [comment]
Unfortunately creating directories in external storage requires SAF permission which is not implemented, sorry for not mentioning in the README file that this is not possible.

**Comment by stavBodik** [comment]
> eating directories in external storage requires SAF permission which

yes but where in the code you hide the image view ? It is an UI bug I think , I tryd to add different items up on : new_folder_button_holder and they are not visible ?

**Comment by stavBodik** [comment]
The real UI bug is that the plus icon is vector drawable and this need to be added in order for it to work 

`android {
    compileSdkVersion 29
    defaultConfig {        
        vectorDrawables.useSupportLibrary = true
    }`

The SAF permissions is another, other related problem . . .

**Comment by stavBodik** [comment]
LOL I found it : 

`if (DiskUtil.isLollipopAndAbove()) {
            mNewFolderImageView.setImageDrawable(ContextCompat.getDrawable(mContext, R.drawable.drawable_plus_to_close));`

**Comment by stavBodik** [comment]
@codekidX 
> Unfortunately creating directories in external storage requires SAF permission which is not implemented, sorry for not mentioning in the README file that this is not possible.

Any way you are right, creating new folder is allowed only in this path : context.getExternalFilesDir(null).getAbsolutePath()

/emulated/0/Android/data/

Even if we add the API 10 request for : android:requestLegacyExternalStorage="true"
